src/components/TooltipTrigger.js

```diff
@@ -59,10 +63,10 @@ const Tooltip = ({ side = 'bottom', type, target: targetId, children, id, delay
       (target.top + target.bottom) / 2 - position.top
     )
     return Utils.switchCase(finalSide,
-      ['top', () => ({ bottom: 0, left, transform: 'rotate(180deg)' })],
-      ['bottom', () => ({ top: 0, left })],
-      ['left', () => ({ right: 0, top, transform: 'rotate(90deg)' })],
-      ['right', () => ({ left: 0, top, transform: 'rotate(270deg)' })]
+      ['top', () => ({ bottom: 1, left, transform: 'rotate(180deg)' })],
```

Comment:
The 1 is to avoid a faint line along the base of the triangle (where it meets up with the tooltip rectangle).

Reply:
nit: can we add a comment to the css block to explain the 1?

Reply:
Done.